src/decide.ts

```diff
@@ -244,7 +246,41 @@ async function assessAndSaveResults(
 	});
 	return assessment;
 }
-
+function calcSizeLoggingNote(size: number): string {
+	const calc = (size: number, convertBytes: boolean) => {
+		return convertBytes
+			? Math.round((size / (1024 * 1024)) * 100) / 100
+			: Math.round((size / 1024) * 100) / 100;
+	};
+	const sizeMB = calc(size, true);
+	const logNote = sizeMB > 1000 ? `${calc(sizeMB, false)}GB` : `${sizeMB}MB`;
+	return logNote;
+}
+function makeDecisionNote(
+	decision: Decision,
+	searchee: Searchee,
+	candidate: Candidate
+): string {
+	const { blockList } = getRuntimeConfig();
+	switch (decision) {
+		case Decision.SIZE_MISMATCH:
+			return ` - (${calcSizeLoggingNote(
+				searchee.length
+			)} -> ${calcSizeLoggingNote(candidate.size)})`;
+		case Decision.RELEASE_GROUP_MISMATCH:
+			return ` - (${searchee.name
+				.match(RELEASE_GROUP_REGEX)?.[0]
+				?.trim()} -> ${candidate.name
+				.match(RELEASE_GROUP_REGEX)?.[0]
+				?.trim()})`;
+		case Decision.BLOCKED_RELEASE:
+			return ` - ("${
+				releaseInBlockList(searchee, blockList) as string
+			}")`;
+		default:
+			return "";
+	}
+}
```

Comment:
any reason not to put this logic inside the switch statement that starts on line 40?

Reply:
If you look at the amount of stuff, it just seemed to me that it would make the actual decision-based messaging switch a lot more cluttered.

There was not really a reason other than that...

Reply:
That seems like a natural place for the logic to go, especially when compared to the fact that we have to re-switch on the same variable in order to do it in this helper function